src/components/Button/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Button';
```

Comment:
이렇게하면 default export아닌가요?  아래처럼해야 named export로 알고 있는뎁
```suggestion
export { Button } from './Button';
```